nuxt.config.ts

```diff
@@ -4,6 +4,13 @@ const resolver = createResolver(import.meta.url)
 
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
+  app: {
+    head: {
+      title: 'firstcommit - find your first commit',
+      charset: 'utf-8',
+      viewport: 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0',
```

Comment:
this is the current value:

```
width=device-width, initial-scale=1
```

Normally it's good to let users zoom in if they need to, for accessibility, although I get that it's inconvenient when safari does it by default. let me think on this.

Reply:
I totally get why you would like to keep it as is, but if that's the case I would suggest changing the font-size value on the input field to prevent the phone from zooming in. I do believe those are the most common fixes for that issue.

Reply:
makes sense. let's make it 16px on mobile then. sound good?

Reply:
Yep, that sounds good to me :)

Reply:
Oh, I see you edited. Yes and no. It's 14px on `[username].vue`. I've added #27 and this could be closed.